cardano-api/cardano-api.cabal

```diff
@@ -15,23 +15,28 @@ extra-source-files:     README.md, ChangeLog.md
 library
   hs-source-dirs:       src
 
-  exposed-modules:      Cardano.Api.Protocol
+  exposed-modules:      Cardano.API
```

Comment:
@Jimbo4350: Could we rename this to `Cardano.Api` for consistency with all of the other modules?

Reply:
We wanted to avoid any downstream changes in this PR. The next PR will make the appropriate changes from `Api` to `API`